tests/integration/RSVP_Tickets_Test.php

```diff
@@ -111,6 +115,7 @@ public function test_ticket_without_stock() {
 		];
 		$rsvp->save_ticket( $post_id, $ticket, $data );
 		$tickets = $rsvp->get_event_tickets( $post_id );
+		codecept_debug( $ticket->inventory() );
```

Comment:
debug

Reply:
Oh snap! thanks for the head's up, Paul. https://github.com/the-events-calendar/event-tickets/pull/1942/files#r573472170

Reply:
I was kind of OK leaving this since it's helpful when debugging anyways, but I'm also OK with removing it :)